src/base_media_base.h

```diff
@@ -107,15 +107,15 @@ struct BaseSet {
 	 * @param isocode the isocode to search for
 	 * @return the description
 	 */
-	const std::string &GetDescription(const std::string &isocode) const
+	const std::string &GetDescription(std::string_view isocode) const
 	{
 		if (!isocode.empty()) {
 			/* First the full ISO code */
-			auto desc = this->description.find(isocode);
+			auto desc = this->description.find(std::string{isocode});
```

Comment:
We have this pattern multiple times by now.
Maybe it is time to enable transparent lookup in `stdafx.h` or somewhere in `core`:
https://godbolt.org/z/jaYe37zf9 (trimmed down example from cppreference)